machine/accton/accton_as7316_54x/busybox/conf/config

```diff
@@ -0,0 +1,12 @@
+CONFIG_SYS_EEPROM=y
+CONFIG_SYS_EEPROM_DEVICE_I2C=y
+CONFIG_SYS_EEPROM_I2C_DEVICE="/dev/i2c-0"
+CONFIG_SYS_EEPROM_I2C_ADDR=0x54
+CONFIG_SYS_EEPROM_I2C_MEM_ADDR_BITS=8
+CONFIG_SYS_EEPROM_OFFSET=0
+CONFIG_SYS_EEPROM_SIZE=512
+CONFIG_SYS_EEPROM_MAX_SIZE=512
+CONFIG_MKSWAP=y
+CONFIG_FEATURE_MKSWAP_UUID=y
+CONFIG_SWAPONOFF=y
+CONFIG_FEATURE_SWAPON_PRI=y
```

Comment:
just holding off until we decide whether the swap commands should be globally applied.  or perhaps applied to x86 busybox?